packages/boiler-task-selenium/src/selenium-standalone/get-selenium-standalone-options.js

```diff
@@ -0,0 +1,15 @@
+export default function getSeleniumStandaloneOptions() {
+  return {
+    installOpts: {
+      version: '2.48.2'
+    },
+    startOptions: {
+      version: '2.53.0',
+      logLevel: 'verbose',
+      spawnOptions: {
+        logLevel: 'verbose',
+        stdio: 'ignore'
+      }
+    }
+  };
+}
```

Comment:
this could just be an `export default` object rather than a function. Also why is the install version different from the start version?